service

```diff
@@ -63,8 +76,9 @@ def main():
 
         print >>sys.stderr, "updated schedule"
 
-        # 6 times 10s (aka 1 minute of only updateing the time but not re-downloading the schdule)
-        for i in xrange(6):
+        # 6*60 times 10s (aka 1 hour of only updating the time but not
+        # re-downloading the schdule)
+        for i in xrange(6*60):  # TODO: ok?
```

Comment:
an hour is awfully long, Is the API really that slow? 
I think 5-10 minutes are more fitting

Reply:
10 minutes should be ok
5 minutes probably too..
change it as you like...